app/assets/javascripts/components/dropdown-menu.js

```diff
@@ -0,0 +1,31 @@
+ManageIQ.angular.app.component('dropdownMenu', {
+  bindings: {
+    id: '<',
+    buttons: '<',
+  },
+  controllerAs: 'vm',
+  controller: function() {
+    var vm = this;
+
+    this.$onInit = function() {
+      vm.dropdown_id = 'btn_' + vm.id;
+      vm.buttons = JSON.parse(vm.buttons);
+    };
+  },
+  template: [
+    '<div class="dropdown pull-right dropdown-kebab-pf">' +
+    '  <button aria-haspopup="true" class="btn btn-link dropdown-toggle" data-toggle="dropdown" id="{{vm.dropdown_id}}" type="button" aria-expanded="false">' +
+    '    <span class="fa fa-ellipsis-v">' +
+    '    </span>' +
+    '  </button>' +
+    '  <ul aria-labelledby="{{vm.dropdown_id}}" class="dropdown-menu dropdown-menu-right">' +
+    '    <li ng-repeat="button in vm.buttons">' +
+    '      <a id="{{button.id}}" title="{{button.title}}" data-method="{{button.dataMethod}}" data-remote="{{button.dataRemote}}" confirm="{{button.confirm}}" href="{{button.href}}" data-miq_spark_on="{{button.sparkOn}}">' +
```

Comment:
data-miq_spark**le**_on ;)

Reply:
Also, neither `sparkOn` nor `sparkleOn` is actually used .. looks like you just skipped them :)